wcfsetup/install/files/lib/acp/form/DevtoolsProjectEditForm.class.php

```diff
@@ -340,7 +340,7 @@ protected function setFormObjectData()
         $instructionsField->value($instructions);
     }
 
-    protected function tooglePackageXmlFieldAvailabilty($available)
```

Comment:
This bug is introduced with WoltLab Suite 3.1. Should we fix them only in `5.4` or should we backport them into `3.1` or should we ignore the typo here?